app/helpers/claims/support/claims_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Claims::Support::ClaimsHelper
```

Comment:
I'm a big believer in that model specific logic, like this, should go into a decorator. Or alternatively a ViewComponent. 
I would also consider renaming this method to `state_tag`, as this doesn't just return something if the claim is in a draft start.